packages/labs/react/src/test/create-component_test.tsx

```diff
@@ -233,104 +235,127 @@ suite('createComponent', () => {
+    assert.equal(el.hidden, false);
+    assert.equal(el.getAttribute('hidden'), wrappedEl.getAttribute('hidden'));
+    assert.equal(el.hidden, wrappedEl.hidden);
+
+    await renderReactComponent({hidden: true});
+    assert.equal(wrappedEl.getAttribute('hidden'), '');
+    assert.equal(wrappedEl.hidden, true);
+    assert.equal(el.getAttribute('hidden'), wrappedEl.getAttribute('hidden'));
+    assert.equal(el.hidden, wrappedEl.hidden);
+
+    await renderReactComponent({hidden: undefined});
+    assert.equal(wrappedEl.getAttribute('hidden'), null);
+    assert.equal(wrappedEl.hidden, false);
+    assert.equal(el.getAttribute('hidden'), wrappedEl.getAttribute('hidden'));
+    assert.equal(el.hidden, wrappedEl.hidden);
+
+    await renderReactComponent({hidden: true});
+    assert.equal(wrappedEl.getAttribute('hidden'), '');
+    assert.equal(wrappedEl.hidden, true);
+    assert.equal(el.getAttribute('hidden'), wrappedEl.getAttribute('hidden'));
+    assert.equal(el.hidden, wrappedEl.hidden);
+
+    await renderReactComponent({hidden: false});
+    assert.equal(wrappedEl.getAttribute('hidden'), null);
+    assert.equal(wrappedEl.hidden, false);
+    assert.equal(el.getAttribute('hidden'), wrappedEl.getAttribute('hidden'));
+    assert.equal(el.hidden, wrappedEl.hidden);
+
+    await renderReactComponent({});
+    assert.equal(el.getAttribute('hidden'), null);
+    assert.equal(el.hidden, false);
+    assert.equal(el.getAttribute('hidden'), wrappedEl.getAttribute('hidden'));
+    assert.equal(el.hidden, wrappedEl.hidden);
   });
 
-  test('can set properties that reflect', async () => {
-    let o = {foo: true};
-    let a = [1, 2, 3];
-    await renderReactComponent({
-      rbool: true,
-      rstr: 'str',
-      rnum: 5,
-      robj: o,
-      rarr: a,
-    });
-    const firstEl = el;
-    assert.equal(el.rbool, true);
-    assert.equal(el.rstr, 'str');
-    assert.equal(el.rnum, 5);
-    assert.deepEqual(el.robj, o);
-    assert.deepEqual(el.rarr, a);
-    assert.equal(el.getAttribute('rbool'), '');
-    assert.equal(el.getAttribute('rstr'), 'str');
-    assert.equal(el.getAttribute('rnum'), '5');
-    assert.equal(el.getAttribute('robj'), '{"foo":true}');
-    assert.equal(el.getAttribute('rarr'), '[1,2,3]');
-    // update
-    o = {foo: false};
-    a = [1, 2, 3, 4];
-    await renderReactComponent({
-      rbool: false,
-      rstr: 'str2',
-      rnum: 10,
-      robj: o,
-      rarr: a,
-    });
-    assert.equal(firstEl, el);
-    assert.equal(el.rbool, false);
-    assert.equal(el.rstr, 'str2');
-    assert.equal(el.rnum, 10);
-    assert.deepEqual(el.robj, o);
-    assert.deepEqual(el.rarr, a);
-    assert.equal(el.getAttribute('rbool'), null);
-    assert.equal(el.getAttribute('rstr'), 'str2');
-    assert.equal(el.getAttribute('rnum'), '10');
-    assert.equal(el.getAttribute('robj'), '{"foo":false}');
-    assert.equal(el.getAttribute('rarr'), '[1,2,3,4]');
+  test('sets boolean aria attributes', async () => {
```

Comment:
It looks like the last commit renamed the wrong test here.

Reply:
fixed